services/npm/npm-rating.tester.js

```diff
@@ -0,0 +1,42 @@
+'use strict'
+
+const { isStarRating } = require('../test-validators')
+const t = (module.exports = require('../tester').createServiceTester())
+
+t.create('score of vue').get('/rating/vue.json').expectBadge({
+  label: 'Rating',
+  message: isStarRating,
+  color: 'brightgreen',
+})
+
+t.create('score of @vue/cli').get('/rating/@vue/cli.json').expectBadge({
```

Comment:
I realize that what you've done here has been done elsewhere in other npm tests, but for awareness the goal with the test title is much more about describing the nature of the test/the scenario; not the package target.

i.e. the scenario here is testing that the npms badge will handle a scoped package, doesn't really matter what that package is